HIP/hip-0000-hts-precompile.md

```diff
@@ -0,0 +1,299 @@
+- hip: <HIP number (this is determined by the HIP editor)>
```

Comment:
Please change the formatting to `---` wrap around the header and remove `-` on each line (as shown in https://raw.githubusercontent.com/hashgraph/hedera-improvement-proposal/master/hip-0000-template.md)

Reply:
done